apps/earn-protocol/components/layout/VaultsListView/VaultsListView.tsx

```
@@ -119,6 +119,26 @@ export const VaultsListView = ({ selectedNetwork, vaultsList }: VaultsListViewPr
     )
   }, [vaultsList])
 
+  const formattedProtocolsSupportedList = useMemo(
+    () =>
+      new Set(
+        vaultsList.reduce(
+          (acc, { arks }) => [
+            // converting a list which looks like `protocolName-token-chainId`
+            // into a unique list of protocols for all vaults
+            ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)